### ISSUE TITLE
List/Tree: Screenreader does not read the role of the buttons properly

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#52682]
**Environment Details:**
VSCode Version : 1.24.0
OS Version : Win10

**Additional Details:**
MAS Violated : MAS 1.3.1
ScreenReader Versions : NVDA2018.1.1

**Repro Steps:**
1. Navigate to Status Bar at the bottom and move to notifications.
2. Move to any notification that appears at the screen.
3. Navigate to any button inside the notification.

**Actual:**
ScreenReader does not read the role as buttons for all the notification buttons. 
Also does not read the name as well as role for close, settings buttons within the notification window.

**Expected:**
The screenreader should read all the buttons along with role as buttons and also should read the name and role for close and settings buttons. Screenreader should read the content info along with first button only.

**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
https://microsoft.sharepoint.com/teams/msenable/mas/Pages/browse-fixes.aspx

**User Impact:**
The screenreader users will not be able to know all the controls present on the screen along with their purpose.

**MAS Reference:**
MAS 1.3.1 - https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

**Attachment For Reference:**
[A11y_VSCode_StatusBar_ScreenReader_Notification.pptx](https://github.com/Microsoft/vscode/files/2127636/A11y_VSCode_StatusBar_ScreenReader_Notification.pptx)

Does this issue occur when all extensions are disabled?: Yes

**Comment by bpasero** [comment]
This seems to be true for anything I focus within a list item.

**Comment by joaomoreno** [comment]
This seems to be an Electron issue. When I copy-paste our HTML to vanilla Chrome, NVDA correctly reads the button roles.

**Comment by bpasero** [comment]
Nice investigation skillz.

**Comment by isidorn** [comment]
This is fixed in electron 3.0. Thus closing since we have adopted to electorn 3.0
